index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+ <div id="main"> 
+  <nav>
+    <div>
+      <h1 id="title">Bjarne Stroustrup</h1>
+      <h3>He is the creator of C++ programming language</h3>
+    </div>
+  </nav>
```

Comment:
Although It works, the ```nav``` tag is intended for major block of navigation links.
for this, you can use a ```header``` tag.
This would help your code to be more concise and semantically correct.